MementoMori/MementoMori.Server/Controllers/DeckController.cs

```diff
@@ -122,5 +124,34 @@ public IActionResult EditDeck(EditedDeckDTO editedDeckDTO)
             }
             return Ok();
         }
+        [HttpPost("createDeck")]
+        public IActionResult CreateDeck(EditedDeckDTO createDeckDTO)
+        {
+            var requesterId = _authService.GetRequesterId(HttpContext);
+            if (requesterId == null)
+                return Unauthorized();
+            try
+            {
+                return Ok(_deckHelper.CreateDeck(createDeckDTO, (Guid)requesterId));
+            }
+            catch (UnauthorizedEditingException ex)
+            {
+                return Unauthorized();
```

Comment:
If I'm not mistaken this will never be triggered.

Reply:
yep